src/js/problems/findAllDuplicates.js

```diff
@@ -2,7 +2,18 @@
 // Orinak: findAllDuplicates([1,1,2,2,2,3,4,5,5,5,5]) ----> [1,2,5]
 
 const findAllDuplicates = arr => {
+    arr.sort()
+    let newArr = [];
+    let res = [];
 
+    for(let i = 0; i < arr.length; i++) {
+        if(arr[i] === arr[i + 1]) {
+            newArr.push(arr[i])
+        }
+    }
+    res = new Set(newArr) 
```

Comment:
xi miangamic ystex ches haytararel res-y?
```suggestion
    const res = new Set(newArr) 
```